_blinkr/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.1.10
```

Comment:
I'm using the same image as the acceptance tests. We've been using the official ruby image for quite some time without any issues.